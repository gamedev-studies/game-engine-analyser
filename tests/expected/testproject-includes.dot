digraph "source tree" {
    overlap=scale;
    size="8,10";
    ratio="fill";
    fontsize="16";
    fontname="Helvetica";
	clusterrank="local";
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/admin.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Node.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Node.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Student-Record-Management-System/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Student-Record-Management-System/loading.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Node_1.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Node_1.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/customer.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/staff.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/admin.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/customer.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/staff.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/customer.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/staff.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Node.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Student-Record-Management-System/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Student-Record-Management-System/adminFunction.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Hashtable.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Node_1.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Tree.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/BST_Node.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/main.cpp" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/admin.h"
	"/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Node.h" -> "/home/ullmann/Documents/research/game-engine-analyser/tests/testproject/Bank-Management/DSAproject/Node_1.h"
}