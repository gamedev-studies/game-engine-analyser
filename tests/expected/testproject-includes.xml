<project>
<file name="./testproject/Bank-Management/DSAproject/main.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/staff.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/customer.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/admin.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/staff.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/main.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/staff.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/customer.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/admin.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/BST_Tree.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/admin.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/BST_Node.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Node.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/staff.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/main.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/staff.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/customer.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/admin.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/Hashtable.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/customer.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/BST_Tree.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Node.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
</file>
<file name="./testproject/Student-Record-Management-System/main.cpp">
<include type="local" name="./testproject/Student-Record-Management-System/loading.h" />
<include type="local" name="./testproject/Student-Record-Management-System/adminFunction.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/Hashtable.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Node_1.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Node.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/Node_1.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/Node_1.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/main.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/staff.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/customer.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/admin.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/Hashtable.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Node_1.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Node.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/Node.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Node_1.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/BST_Tree.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Node.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/admin.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/customer.h">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
</file>
<file name="./testproject/Student-Record-Management-System/main.cpp">
<include type="local" name="./testproject/Student-Record-Management-System/loading.h" />
<include type="local" name="./testproject/Student-Record-Management-System/adminFunction.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/BST_Tree.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
</file>
<file name="./testproject/Bank-Management/DSAproject/main.cpp">
<include type="local" name="./testproject/Bank-Management/DSAproject/BST_Tree.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/staff.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/customer.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/Hashtable.h" />
<include type="local" name="./testproject/Bank-Management/DSAproject/admin.h" />
</file>
</project>
